ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -330,27 +338,23 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
     #[allow(clippy::too_many_arguments)]
     pub fn verify_opcode_proof(
         &self,
-        _name: &str,
+        name: &str,
         circuit_vk: &VerifyingKey<E>,
-        proof: &ZKVMOpcodeProof<E>,
+        proof: &ZKVMChipProof<E>,
         num_instances: usize,
         pi: &[E],
         transcript: &mut impl Transcript<E>,
         num_product_fanin: usize,
         _out_evals: &PointAndEval<E>,
         challenges: &[E; 2], // derive challenge from PCS
-    ) -> Result<Point<E>, ZKVMError> {
+    ) -> Result<(), ZKVMError> {
         let cs = circuit_vk.get_cs();
         let (r_counts_per_instance, w_counts_per_instance, lk_counts_per_instance) = (
             cs.r_expressions.len(),
             cs.w_expressions.len(),
             cs.lk_expressions.len(),
         );
-        let (log2_r_count, log2_w_count, log2_lk_count) = (
-            ceil_log2(r_counts_per_instance),
-            ceil_log2(w_counts_per_instance),
-            ceil_log2(lk_counts_per_instance),
-        );
+        let num_records = r_counts_per_instance + w_counts_per_instance + lk_counts_per_instance;
```

Comment:
`num_batched` might be better name

Reply:
addressed in https://github.com/scroll-tech/ceno/pull/942/commits/08a7143372bccdbba6cc528b65a0f5f74bc63fce.